### ISSUE TITLE
Consistent naming for PURL

### ISSUE THREAD

**Issue by tsteenbe** [spdx/spdx-3-model#493]
We use packageUrl in 1) and pkgUrl in 2) which for me is inconsistent.

1) https://github.com/spdx/spdx-3-model/blob/main/model/Software/Properties/packageUrl.md?plain=1  A packageUrl (commonly pronounced and referred to as "purl")
2)https://github.com/spdx/spdx-3-model/blob/main/model/Core/Vocabularies/ExternalIdentifierType.md#entries states pkgUrl: https://github.com/package-url/purl-spec

**Comment by jeff-schutt** [comment]
@iamwillbar do you have a strong preference on using "purl" or "packageurl"? 

Note that SPDX version 2 uses [purl](https://spdx.github.io/spdx-spec/v2.3/external-repository-identifiers/#f35-purl)

**Comment by jeff-schutt** [comment]
CC @goneall @rnjudge

**Comment by jeff-schutt** [comment]
Security working group discussed in 9/27 meeting and agreed consistency in the naming is most important. 

We suggest changing the property name in the Package Element from "packageurl" to "purl".

**Comment by goneall** [comment]
@sbarnum @zvr - is there an issue in RDF if we have the same URI representing the individual and a property?

**Comment by rnjudge** [comment]
In the November 28th tech call we decided to update "purl" to "packageUrl". I will open a PR to reflect this decision.

**Comment by henkbirkholz** [comment]
The initial issue was about `packageUrl` vs. `pkgUrl`?

If you are using text encoding already, it does not really matter how verbose you are already: `packageUrl` sounds fine.

**Comment by rnjudge** [comment]
> The initial issue was about `packageUrl` vs. `pkgUrl`?
> 
> If you are using text encoding already, it does not really matter how verbose you are already: `packageUrl` sounds fine.

@henkbirkholz Yes to the initial issue but we use `purl` in [spdx 2](https://github.com/spdx/spdx-spec/blob/8a595028b0386138c84f7188237787c374f6a6cf/chapters/external-repository-identifiers.md?plain=1#L181) which was also being considered for consistency but packageUrl was the final decision.
